registers/TUTORIAL.md

```diff
@@ -14,3 +14,45 @@ You can run it with
 
 By default it uses the "register" register (a list of all registers)
 but you can point it at others with the ```--register``` flag.
+
+Step 2
+------
+
+Next, we are going to put the data into a Trillian log. The first step
+is run the log server and create a log.
+
+Build Trillian:
+
+```make trillian```
+
+Run the log server (this will tie up a terminal window):
+
+```make tlserver```
+
+And create a log (if you need to redo this, you'll need to delete
+logid):
```

Comment:
"to delete the `logid` file" - I misunderstood on the first read and assumed you meant the Trillian tree with the associated ID.

Reply:
Well, you should probably also do that, but I don't know how - so ... how?

Reply:
There's a `DeleteTree` RPC command on the AdminAPI which should do the trick.

Reply:
See https://github.com/google/trillian/pull/983.

Reply:
Nicely Nicelington!